sftp/2.4/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc

```diff
@@ -199,11 +199,7 @@ If you face SSH connection issues due to unsupported ciphers, macs, hostKeyAlgor
 
 === Override Standard Write Strategy
 
-If your *File Write* operation overrides instead of appending when using the `APPEND` mode, use the `CUSTOM` write strategy as it isn’t applicable to the `OVERWRITE` or `CREATE_NEW` modes. With this strategy, you can manually specify the offset for the server to write the file in `APPEND` mode, particularly only for older SFTP servers that don’t directly support write modes. Use this strategy only if the `STANDARD` strategy results in errors during file writing in `APPEND` mode and doesn’t offer concurrency. Additionally, you can adjust the buffer size in the *Buffer Size for Write Strategy* field. 
-
-=== Override Max Idle Configuration
-
-If you use a custom write strategy and experience consecutive write call failures during the `APPEND` write mode, set the *Max Idle* field in the Advanced tab of the SFTP Connection Config to `0`. This change forces the SFTP connection to be recreated for each connection, ensuring that the `APPEND` write mode executes successfully. This workaround incurs a performance impact due to the overhead of frequent connection reestablishment.
+If your *File Write* operation overrides instead of appending when using the `APPEND` mode, use the `CUSTOM` write strategy along with the `CUSTOM_APPEND` mode only as it isn't applicable to the other modes. With this strategy, you can manually specify the offset for the server to write the file in `APPEND` mode, particularly only for older SFTP servers that don't directly support write modes. Use this strategy only if the `STANDARD` strategy results in errors during file writing in `APPEND` mode and doesn’t offer concurrency. Additionally, you can adjust the buffer size in the *Buffer Size for Write Strategy* field. 
```

Comment:
```suggestion
If your *File Write* operation overrides the `max idle` configuration instead of appending it when using the `APPEND` mode, use the `CUSTOM` write strategy along with the `CUSTOM_APPEND` mode. This strategy isn't applicable to the other modes. With this strategy, you can manually specify the offset for the server to write the file in `APPEND` mode. This strategy is useful for older SFTP servers that don't directly support the `WRITE` mode. Use this strategy only if the `STANDARD` strategy results in errors during file writing in `APPEND` mode and doesn’t offer concurrency. Additionally, adjust the buffer size in the *Buffer Size for Write Strategy* field. 
```

Reply:
Please, review and ensure that my changes are appropriate.